cmd/xl-v1-multipart.go

```diff
@@ -446,17 +418,12 @@ func (xl xlObjects) PutObjectPart(ctx context.Context, bucket, object, uploadID
 		return pi, toObjectErr(err, minioMetaMultipartBucket, uploadIDPath)
 	}
 
-	fi, err := xl.statPart(ctx, bucket, object, uploadID, partSuffix)
-	if err != nil {
-		return pi, toObjectErr(err, minioMetaMultipartBucket, partSuffix)
-	}
-
 	// Return success.
 	return PartInfo{
 		PartNumber:   partID,
-		LastModified: fi.ModTime,
+		LastModified: xlMeta.Stat.ModTime,
 		ETag:         md5hex,
```

Comment:
Note ListParts won't return different modTimes per part, this code change is going to make sure that for a given uploadId we send same modTimes for all parts. 

We haven't seen any application which uses this information in any meaningful way, we also don't save this information for future retrieval. It makes sense to drop this support.